docs/getting-started-guides/ubuntu.md

```diff
@@ -118,7 +118,7 @@ The `FLANNEL_NET` variable defines the IP range used for flannel overlay network
 
 After all the above variables being set correctly, we can use following command in cluster/ directory to bring up the whole cluster.
 
-`$ KUBERNETES_PROVIDER=ubuntu ./kube-up.sh`
+`$ KUBERNETES_PROVIDER=ubuntu ../kube-up.sh`
```

Comment:
The maintainer of the getting started guide pointed out to me that just before the invocation of kube-up.sh, this sentence appears:

After all the above variables being set correctly, we can use following command in cluster/ directory to bring up the whole cluster.

I'm sorry I missed that sentence.

I'm willing to close this issue as essentially wontfix.